components/webui/about_ui.cc

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/webui/about_ui.h"
+#include <string_view>
```

Comment:
new line

Reply:
should be:
```
#include "brave/components/webui/about_ui.h"

#include <string_view>

#include "base/containers/contains.h"
```

Reply:
Fixed in https://github.com/brave/brave-core/pull/20929/commits/28844bde3166f309f9a3e7c9b72c59646c65d4fa